README.md

```diff
@@ -170,6 +170,41 @@ The following example shows how to use it in a Kafka Consumer consuming from Apa
     ssl.truststore.certificates=${secrets:myproject/my-cluster-cluster-ca-cert:ca.crt}
     ```
 
+## Using patterns
+
+You can also use [Glob patterns](https://en.wikipedia.org/wiki/Glob_(programming)) to specify the keys in the Secret or Config Map that should be used.
+The Kubernetes Config Provider will find all fields matching the pattern and join them together into some value.
```

Comment:
"into some value"? "some" sounds to me to be something indeterminate ... what did you mean?

Reply:
I changed it to single -> `into a single value` ... that should make sense here